roles/ceph-rbd-mirror/tasks/pre_requisite.yml

```diff
@@ -9,7 +9,13 @@
     - package-install
 
 - name: create rbd-mirror keyring
-  command: ceph --cluster {{ cluster }} --name client.bootstrap-rbd --keyring /var/lib/ceph/bootstrap-rbd/{{ cluster }}.keyring auth get-or-create client.rbd-mirror.{{ ansible_hostname }} mon 'profile rbd' osd 'profile rbd' -o /etc/ceph/{{ cluster }}.client.rbd-mirror.{{ ansible_hostname }}.keyring
+  command: >
+    ceph --cluster {{ cluster }}
+    --name client.bootstrap-rbd-mirror
+    --keyring /var/lib/ceph/bootstrap-rbd-mirror/{{ cluster }}.keyring
+    auth get-or-create client.rbd-mirror.{{ ansible_hostname }}
+    mon 'profile rbd-mirror'
```

Comment:
This is missing the "osd 'profile rbd'" line

Reply:
Done, thanks